src/ORM/Connect/DBQueryBuilder.php

```diff
@@ -68,13 +68,23 @@ public function buildSQL(SQLExpression $query, &$parameters)
      */
     protected function buildSelectQuery(SQLSelect $query, array &$parameters)
     {
-        $sql  = $this->buildSelectFragment($query, $parameters);
+        $needsParenthisis = count($query->getUnions()) > 0;
+        $nl = $this->getSeparator();
```

Comment:
Could you rename variable please, probably just to `$seperator` (or `$newline`?) I'm not sure what it's supposed to be

Reply:
Can do - though I'll point out that `$nl` is used all throughout this file. I'm not going to change the other usages of that variable.

Reply:
Done - though I think it makes the file overall less readable due to the introduced inconsistency.

Reply:
Ah ok, just revert it to `$nl` then

Reply:
Reverted